packages/qwik/src/optimizer/src/plugins/vite-server.ts

```diff
@@ -153,22 +153,22 @@ export async function configureDevServer(
 
 export async function configurePreviewServer(
   middlewares: Connect.Server,
-  opts: NormalizedQwikPluginOptions,
+  ssrOutDir: string,
```

Comment:
question: what is the reason not to pass all the other options and to replace it with only this one?

thanks

Reply:
`ssrOutDir` is not present within the options object, `opts` themselves were used only to grab `opts.rootDir`, which is not longer needed. 

And it doesn't feel reasonable to include `ssrOutDir` into the `NormalizedQwikPluginOptions`, because it already contains `outDir` property, which is resolved either as `clientOutDir` or `ssrOutDir`

Reply:
thanks @dmitry-stepanenko !